### DISCUSSION TITLE
iOS 上设置了 setRepeatCount 1 的情况下，不规律出现播放两次动画

### DISCUSSION THREAD

**Discussion by iTofu** [Tencent/libpag#999] [Q&A]
pod 版本：
- libpag (4.2.92)

PAG 文件：
https://github.com/iTofu/ImageHosting/blob/master/20230620/B057FEE3-5907-442F-9A97-00819D55A2D6.pag

不知道是不是 pag 文件参数的问题，麻烦有空的话帮我们看一下，谢谢！

**Comment by domchen** [comment]
麻烦提供一下可以复现的demo，这样才能帮你解决问题。

**Reply by iTofu** [reply]
后续暂未复现哈，在持续观察中
